resources/leiningen/new/xiana/src/backend/app/interceptors.clj

```diff
@@ -1,14 +0,0 @@
-(ns interceptors
-  (:require
-    [potemkin :refer [import-vars]]
-    [xiana.core :as xiana]))
-
-(comment
-  (import-vars
-    [framework.components.app.interceptors
-     sample-router-interceptor
-     sample-controller-interceptor]))
-
-(def sample-{{sanitized-name}}-controller-interceptor
-  {:enter (fn [{request :request {:keys [handler controller match]} :request-data :as state}]
-            (xiana/ok state))})
```

Comment:
I don't know, maybe good to have a sample interceptor here, but maybe it should be extended with a :leave fn

Reply:
https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/src/backend/app/interceptors.clj#L5-L9